codegen/src/types/derives.rs

```diff
@@ -54,6 +56,13 @@ impl DerivesRegistry {
         }
         Derives { derives: defaults }
     }
+
+    pub fn default_with_crate_path(crate_path: &CratePath) -> Self {
+        Self {
+            default_derives: Derives::default_with_crate_path(crate_path),
```

Comment:
How about removing the `Default` impls because we never want to be able to forget to provide a crate path?

This could then just be `new` or `new_with_crate_path`